src/client/src/apps/SimpleLauncher/spotlight/components/InlineBlotter.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react'
```

Comment:
this has been refactored and moved from `src/client/src/apps/SpotlightRoute/InlineBlotter.tsx`. seems that github does not pick this up